custom_model_runner/datarobot_drum/resource/drum_server_utils.py

```diff
@@ -7,7 +7,7 @@
 
```

Comment:
someone definitely chime in if I shouldn't have moved this and the utils file directly below into drum resources

Reply:
Ok.... Agreed..
Could you just add description to the classes and methods in `drum_server_utils` and `utils` files.
E.g. `DrumServerRun` helper class used in validation and tests to start server as a thread... etc.